numpy/random/mtrand/mtrand.pyx

```diff
@@ -4460,8 +4460,8 @@ cdef class RandomState:
 
         A loaded dice is more likely to land on number 6:
 
-        >>> np.random.multinomial(100, [1/7.]*5)
-        array([13, 16, 13, 16, 42])
+        >>> np.random.multinomial(100, [1/7.]*6)
```

Comment:
Yes, a bit more explanation of the example would help. I confess to liking the R convention better ;)